tools/deployment/onnx2tensorrt.py

```diff
@@ -22,6 +24,8 @@ def onnx2tensorrt(onnx_file,
         trt_file (str): Filename of the output TensorRT engine file.
         input_shape (list[int]): Input shape of the model.
             eg [1, 3, 224, 224].
+        max_shape (list[int]): Max input shape of the model.
```

Comment:
The `max_shape` in docstring does not match any input parameter. You may fix this and specify how max_shape is obtained by setting max_batch_size